sdk/eventhub/event-hubs/test/hubruntime.spec.ts

```diff
@@ -57,10 +57,26 @@ describe("RuntimeInformation #RunnableInBrowser", function(): void {
         err.message.should.match(/The [\w]+ operation has been cancelled by the user.$/gi);
       }
     });
+
+    it("can be ran in parallel without retries", async function (): Promise<void> {
+      client = new EventHubClient(service.connectionString, service.path, {
+        retryOptions: {
+          maxRetries: 0
+        }
+      });
+      const results = await Promise.all([
+        client.getPartitionIds(),
+        client.getPartitionIds()
+      ]);
+
+      for (const result of results) {
+        result.should.have.members(arrayOfIncreasingNumbersFromZero(result.length));
+      }
+    });
   });
 
-  describe("hub runtime information", function(): void {
-    it("gets the hub runtime information", async function(): Promise<void> {
+  describe("hub runtime information", function (): void {
```

Comment:
I ran npm format but must have made another change that autoformatted with different settings. Will rerun npm format.